src/pages/index.js

```diff
@@ -13,7 +30,10 @@ export default class IndexPage extends React.Component {
         <section className="section">
           <div className="container">
             <div className="content">
-              <img src={pic} alt="harry" />
+              <img
+								onMouseEnter={this.togglePic} onMouseLeave={this.togglePic}
```

Comment:
This is a little buggy, see https://deploy-preview-1--harry-style.netlify.com/

maybe try onMouseOver and onMouseOut